src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Rules/Operations/IndentBlockOperation.cs

```diff
@@ -10,9 +10,25 @@ namespace Microsoft.CodeAnalysis.Formatting.Rules
     /// <summary>
     /// set indentation level for the given text span. it can be relative, absolute or dependent to other tokens
     /// </summary>
-    internal sealed class IndentBlockOperation
+    internal readonly struct IndentBlockOperation
```

Comment:
📝 This is a very large structure. We need to demonstrate substantial improvements for conversion to a value type before making this change.

Reply:
i disagree.  i think it would be sufficient to show no performance regressions here.